css-four-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Barlow', sans-serif;
+}
+
+.container {
+  background-color: rgb(247, 248, 250);
+  justify-content: center;
+  align-items: center;
+  margin: auto;
```

Comment:
These three declarations can be removed, they aren't doing anything right now, since there is no extra margin to spread with `auto` and this element is not `display: flex`, so the flexbox properties will do nothing.